### PR TITLE
fix: update to correct text colors in dark theme

### PR DESCRIPTION
**Pull request by adriansberg**

### CODE REVIEW

File: packages/assets/files/fram/colors/icons/favicon.svg [hunk 1]
```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" fill="none"><g clip-path="url(#a)"><path fill="#005683" stroke="#1D1D1B" stroke-miterlimit="10" stroke-width=".536" d="M20 0H0v20h20V0Z"/><path fill="#fff" fill-rule="evenodd" d="M15.993 2.281H4.008c-.144 5.723-.357 10.627 5.993 15.432 6.35-4.803 6.136-9.707 5.992-15.432ZM5.036 9.521a2.645 2.645 0 0 0 1.92-2.542V4.646H5.403v-.388h1.553V3.146h.388v1.112h1.553v.388H7.344V6.98c0 1.208.813 2.228 1.92 2.542a2.115 2.115 0 1 1-4.228 0Zm7.064 2.536a2.112 2.112 0 1 1-4.226 0v-.002a2.643 2.643 0 0 0 1.92-2.543V7.18H8.24V6.79h1.553v-1.11h.388v1.112h1.553v.389h-1.553v2.332a2.64 2.64 0 0 0 1.918 2.543Zm2.836-2.536a2.113 2.113 0 1 1-4.226 0 2.645 2.645 0 0 0 1.92-2.542V4.646h-1.555v-.388h1.553V3.146h.388v1.112h1.555v.388h-1.553V6.98a2.641 2.641 0 0 0 1.918 2.542Z" clip-rule="evenodd"/></g><defs><clipPath id="a"><path fill="#fff" d="M0 0h20v20H0z"/></clipPath></defs></svg>
```
Comment (mortennordseth) [inline packages/assets/files/fram/colors/icons/favicon.svg:1]: I don't think we should edit the FRAM logo 🤔

**Review by mortennordseth — CHANGES_REQUESTED** [review]

File: packages/assets/files/fram/colors/icons/favicon.svg (hunk 1 above)
Comment (adriansberg) [reply packages/assets/files/fram/colors/icons/favicon.svg:1]: Hmm, sure

Comment (mortennordseth) [reply packages/assets/files/fram/colors/icons/favicon.svg]: Ref. this: 

image

**Pull request merged**